source/client/service_impl.h

```diff
@@ -35,9 +35,15 @@ class ServiceImpl final : public nighthawk::client::NighthawkService::Service,
   /**
    * Constructs a new ServiceImpl instance
    */
-  ServiceImpl() : process_wide_(std::make_shared<Envoy::ProcessWide>()) {
-    logging_context_ = std::make_unique<Envoy::Logger::Context>(
-        spdlog::level::from_str("info"), "[%T.%f][%t][%L] %v", log_lock_, false);
+  ServiceImpl(std::unique_ptr<Envoy::OptionsImpl> options = nullptr)
```

Comment:
Wait .. one ask, does this then scan/parse the CLI for arguments that match Envoy's CLI in nighthawk_service? If so, wouldn't it be better to replicate these options in Nighthawk's own?